libraries/apollo-ast/src/main/resources/builtins.graphqls

```diff
@@ -1,6 +1,6 @@
 # Built in scalar and introspection types from:
 # - https://spec.graphql.org/June2018/#sec-Scalars
-# - https://spec.graphql.org/June2018/#sec-Schema-Introspection
+# - https://github.com/graphql/graphql-spec/blob/afc0a35d271ba9502c3c68aeda6e6c6fbc223774/spec/Section%204%20--%20Introspection.md
```

Comment:
Can you add a comment here ? 

```
# In theory the user needs to provide the builtin definitions because we cannot know in advance what
# version of the spec they are using neither if they have extended any of the introspection types.
# This file is a fallback to make a best effort guess in case the user didn't provide these definitions at the 
# risk of potentially validating invalid queries.
```